concepticondata/conceptlists/Josserand-1983-188.tsv

```diff
@@ -0,0 +1,189 @@
+ID	NUMBER	SPANISH	ENGLISH	CONCEPTICON_ID	CONCEPTICON_GLOSS
+Josserand-1983-188-1	1	lluvia	rain	658	RAIN (PRECIPITATION)
+Josserand-1983-188-2	2	maguey	agave		
+Josserand-1983-188-3	3	petate	straw mat	195	MAT
+Josserand-1983-188-4	4	cuatro	four	1500	FOUR
+Josserand-1983-188-5	5	pluma	feather	1201	FEATHER
+Josserand-1983-188-6	6	cola	tail	1220	TAIL
+Josserand-1983-188-7	7	cera	wax	1931	BEESWAX
+Josserand-1983-188-8	8	nariz	nose	1221	NOSE
+Josserand-1983-188-9	9	palabra	word	1599	WORD
+Josserand-1983-188-10	10	está cerca	near	1942	NEAR
+Josserand-1983-188-11	11	árbol	tree	906	TREE
+Josserand-1983-188-12	12	venado	deer	1936	DEER
+Josserand-1983-188-13	13	manteca	lard	1282	FAT (FROM ANIMALS)
+Josserand-1983-188-14	14	olor	smell	2124	SMELL
+Josserand-1983-188-15	15	tortilla	tortilla		
+Josserand-1983-188-16	16	vomita	vomits	1278	VOMIT
+Josserand-1983-188-17	17	flojo	lazy	1564	LAZY
+Josserand-1983-188-18	18	cama	bed	1663	BED
+Josserand-1983-188-19	19	comal	griddle	31	PAN
```

Comment:
I wasn't sure about that one. I guess it would be nice to have a more specific concept, the 'comal' is really a specific type of pan, often used over open fire.

Reply:
If it is a very regional type of pan, I'd suggest to unmap it.